win32/tests/srw_lock_win32_ut/CMakeLists.txt

```diff
@@ -18,5 +18,5 @@ set(${theseTestsName}_cpp_files
 set(${theseTestsName}_h_files
 )
 
-build_test_artifacts(${theseTestsName} "tests/c_pal/win32" ADDITIONAL_LIBS pal_interfaces c_pal_reals)
+build_test_artifacts(${theseTestsName} "tests/c_pal/win32" ADDITIONAL_LIBS pal_interfaces c_pal_reals pal_ll_interfaces pal_ll_win32 c_pal_ll_reals pal_interfaces_reals)
```

Comment:
>pal_ll_win32 [](http://example.com/codeflow?start=120&length=12)

is this needed? it's a unit test so I would think we only need to link to the xx_reals libs

Reply:
removed